### PR TITLE
[LDN-2019] Novel Coronavirus Information + Sponsor

### PR DESCRIPTION
**Pull request by chriscoffee**
- update welcome page and sponsor page to show statement on covid-19
- add gold sponsor circleci who have paid.

### CODE REVIEW

File: content/events/2020-london/welcome.md [hunk 1]
```diff
@@ -5,8 +5,17 @@ aliases = ["/events/2020-london/"]
 Description = "devopsdays London 2020"
 +++
 
-<div style="text-align:center;">
-  {{< event_logo >}}
+<div class="row">
+  <div class="col-md-4">
+    <img alt="DevOpsDays London 2019" src="/events/2020-london/logo.png" class="img-fluid">
```
Comment (bridgetkromhout) [inline content/events/2020-london/welcome.md:10]: alt text references 2019

File: content/events/2020-london/sponsor.md [hunk 2]
```diff
@@ -4,9 +4,14 @@ Type = "event"
 Description = "Sponsor devopsdays London 2020"
 +++
 
+<div class="col-md">
+  <div class="alert alert-warning" role="alert">
+    Notice to sponsors regarding novel coronavirus (COVID-19)
+  </div>
+  <p>Please note that paid sponsorship is non-refundable. As this is an unpaid volunteer community-run event, we appreciate your understanding. We use all the collected funds to put on the event, and we are committed to paying all those monies to our venue, food, AV, and other suppliers. You can request to be put on a refund list and if we sell out and can resell your slot we will refund you. In the case of a major event (like the conference being forced to cancel due to COVID-19) we will attempt to recoup costs from our suppliers and may be able to offer full or partial refunds, but that is us doing our resonable efforts; we are unfortunately unable to commit to it.</p>
+</div>
```
Comment (bridgetkromhout) [inline content/events/2020-london/sponsor.md:12]: spelling - "resonable" = "reasonable"?

Comment (chriscoffee) [reply content/events/2020-london/sponsor.md:12]: i should really enable spell again on vim 👍 thanks for this!!

**Comment by chriscoffee** [comment]
Thanks for the review @bridgetkromhout, much appreciated! I've corrected those issues so should be good to go now 😌

**Pull request merged**